mjpc/task.h

```diff
@@ -68,12 +68,13 @@ class Task {
 
   // cost parameters
   int num_residual;
-  int num_norms;
+  int num_cost;
+  int num_trace;
   std::vector<int> dim_norm_residual;
-  std::vector<int> num_norm_parameters;
+  std::vector<int> num_num_parameter;
```

Comment:
Can you add a comment on what this number is? Weird name.

Reply:
Bad find-and-replace... Fixed to `num_norm_parameter`.